Gems/ROS2/gem.json

```diff
@@ -18,8 +18,8 @@
         "ROS2"
     ],
     "compatible_engines": [
-        "o3de-sdk>=2.1.0",
-        "o3de>=2.1.0"
+        "o3de-sdk>=4.1.0",
+        "o3de>=4.1.0"
```

Comment:
Shouldn't this be '3'?

Reply:
Set to 3.1.0. Thanks!

Reply:
Sorry, I just checked o3de/development and the engine version is [4.0.0](https://github.com/o3de/o3de/blob/57c811f7fb5505600b081f14bf197b3ce1c26183/engine.json#L7), sorry about that.